src/as-relation.c

```diff
@@ -1044,6 +1184,16 @@ as_relation_load_from_xml (AsRelation *relation, AsContext *ctx, xmlNode *node,
 		g_autofree gchar *side_str = (gchar*) xmlGetProp (node, (xmlChar*) "side");
 		priv->display_side_kind = as_display_side_kind_from_string (side_str);
 
+		g_free (priv->version);
```

Comment:
I know it's not your code, but what about `g_free (g_steal_pointer (&priv->version))` or `as_relation_set_version (relation, NULL)`?